sessionctx/variable/session.go

```diff
@@ -2110,7 +2110,12 @@ func (s *SessionVars) EncodeSessionStates(ctx context.Context, sessionStates *se
 	}
 	sessionStates.LastFoundRows = s.LastFoundRows
 	sessionStates.SequenceLatestValues = s.SequenceState.GetAllStates()
-	sessionStates.MPPStoreLastFailTime = s.MPPStoreLastFailTime
+	sessionStates.MPPStoreLastFailTime = make(map[string]time.Time, 0)
```

Comment:
I think it's fine.
`EncodeSessionStates` is called only when migrating sessions. What's more, it will marshal `MPPStoreLastFailTime`, which may cost more time than copying. I think the size of `MPPStoreLastFailTime` won't be very big, so it's fine.